config/config.md

```diff
@@ -44,6 +44,10 @@ The following config option are provided by the OpenHIM. All of these options ha
     // The session secret key used for the hashing of signed cookie (used to detect if the client modified the cookie)
     // Signed cookie is another cookie of the same name with the .sig suffix appended
     "sessionKey": "r8q,+&1LM3)CD*zAGpx1xm{NeQhc;#",
+    // If OpenHIM is behind a proxy (should be `true` if the ssl is handled by the proxy)
```

Comment:
```suggestion
    // If OpenHIM is behind a proxy (should be `true` if the proxy sends relevant Forwarded headers)
```
Just updated the wording a bit since it is not related solely to ssl but more being behind a proxy in general